src/main/java/org/springframework/data/elasticsearch/repository/query/ElasticsearchQueryMethod.java

```diff
@@ -69,9 +69,9 @@
  */
 public class ElasticsearchQueryMethod extends QueryMethod {
 
-	// the following 2 variables exits in the base class, but are private. We need them for
+	// the following 2 variables exists in the base class, but are private. We need them for
 	// correct handling of return types (SearchHits), so we have our own values here.
-	// Alas this means that we have to copy code that initializes these variables and in the
+	// Also, this means that we have to copy code that initializes these variables and in the
```

Comment:
```suggestion
	// This means that we have to copy code that initializes these variables and in the
```
get rid of that word